elixir/apps/api/mix.exs

```diff
@@ -4,7 +4,7 @@ defmodule API.MixProject do
   def project do
     [
       app: :api,
-      version: String.trim(File.read!("../../VERSION")),
+      version: "0.1.0",
```

Comment:
We should read the actual version here. It's used in logging and error reporting. We can read the commit sha whenever the artifacts are built. (Keep in mind `.git` is added to `.gitingore`).

Reply:
Fixed. We now populate a file, `GIT_SHA` during build with the git sha, and read that at runtime inside the app